<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-size: 14px;
}

QListWidget {
border: 2px solid gray;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: rgb(60, 60, 60);
color: white;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
font-size: 22px;
}


QTextEdit {
border: 2px solid gray;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: rgb(60, 60, 60);
color: white;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
}

QLabel {
background: none;
color: white;
}

QProgressBar {
color: white;
border: 2px solid  rgb(80, 80, 80);
text-align: center;
padding: 1px;
border-radius: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
width: 30px;
height: 14px; 
}

QProgressBar::chunk {
border-radius: 3px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0.0 #fff, stop: 0.4999 #8f0,  stop: 0.5 #8f0, stop: 1.0 #050);
border: 1px solid rgb(0, 0, 0);
}

QGroupBox {
border: 2px solid;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
padding-top: 12px;
color: white;
selection-color: rgb(30, 30, 30);
}

QPushButton {
color: white;
border: 2px solid rgb(80, 80, 80);
border-radius: 5px;
padding: -2px;

background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
min-width: 80px;
min-height:30px;
}

QPushButton:hover {
border: 2px solid  rgb(160, 160, 160);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #fff, stop: 1 #111);
}

 QPushButton:pressed {
color:  rgb(85, 255, 0);
border: 2px solid rgb(85, 255, 0);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #fff, stop: 1 #111);
}

QMenu{
border: 2px solid rgb(85, 255, 0);
border-radius: 5px;
font-size: 22px;
min-height:25px;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
}






QScrollBar:horizontal {
border: 2px solid;
border-color: rgb(85,255,0);
border-radius: 5px;
background: rgb(60,60,60);
height: 25px;
margin: 2px 2px 2px 2px;
}
QScrollBar::handle:horizontal {
background: rgb(255,255,255);
min-width: 30px;
}
QScrollBar::add-line:horizontal {
background: none;
width: 20px;
subcontrol-position: right;
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
background: none;
width: 20px;
subcontrol-position: left;
subcontrol-origin: margin;
}
QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
width: 3px;
height: 3px;
background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
background: none;
}



QScrollBar:vertical {
border: 2px solid;
border-color: rgb(85,255,0);
border-radius: 5px;
background: rgb(60,60,60);
width: 25px;
margin: 2px 2px 2px 2px;
}
QScrollBar::handle:vertical {
background: rgb(255,255,255);
min-height: 30px;
}
QScrollBar::add-line:vertical {
background: none;
height: 20px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
background: none;
height: 20px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:vertical {
width: 3px;
height: 3px;
background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:vertical {
background: none;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Switcher [v1.0]</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="RCQListWidget" name="listWidgetTasks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Menu</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonTaskinfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>System Info</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonExit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonSettings">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RCQListWidget</class>
   <extends>QListWidget</extends>
   <header>rcqlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
